types/superagent-prefix/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for superagent-prefix 0.0
+// Project: https://github.com/johntron/superagent-prefix
+// Definitions by: Michael Ledin <https://github.com/mxl/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as request from 'superagent';
+
+export default function(prefix: string): request.Plugin;
```

Comment:
An `export =` should be matched by an `import =`, as in `import plugin = require('superagent-no-cache');`.
You could also add an `export namespace plugin {}` to make the `import *` work...